.github/workflows/build.yml

```diff
@@ -98,6 +106,126 @@ jobs:
+
+          # Build the amd64 snap
+          cp ./dist/linux-amd64_linux_amd64_v1/parca-agent snap/local/parca-agent
+          snapcraft pack --verbose --build-for amd64
+
+          # Build the arm64 snap
+          cp ./dist/linux-arm64_linux_arm64/parca-agent snap/local/parca-agent
+          snapcraft pack --verbose --build-for arm64
+
+      - name: Upload locally built snap artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: built-snaps
+          path: |
+            *.snap
+
+  test-snap:
+    name: Test Snap
+    needs: snap
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch built snap
+        uses: actions/download-artifact@v4
+        with:
+          name: built-snaps
+
+      - name: Install snap & invoke Parca Agent
+        run: |
+          sudo snap install --classic --dangerous *_amd64.snap
+
+          sudo snap set parca-agent log-level=debug
+          parca-agent --help
+
+      - name: Start Parca Agent - default config
+        run: |
+          sudo snap start parca-agent
+
+          # Set some options to allow retries while Parca Agent comes back up
+          CURL_OPTS=(--max-time 20 --retry 5 --retry-delay 3 --retry-connrefused)
+
+          curl ${CURL_OPTS[@]} http://localhost:7071/
+          curl ${CURL_OPTS[@]} http://localhost:7071/metrics
+
+      - name: Configure snap - node name
+        run: |
+          sudo snap set parca-agent node=foobar
+          sudo snap restart parca-agent
+
+          # Set some options to allow retries while Parca Agent comes back up
+          CURL_OPTS=(--max-time 20 --retry 5 --retry-delay 3 --retry-connrefused)
+
+          curl ${CURL_OPTS[@]} http://localhost:7071/
+          curl ${CURL_OPTS[@]} http://localhost:7071/metrics
+
+      - name: Configure snap - http address
+        run: |
+          sudo snap set parca-agent http-address=":8081"
+          sudo snap restart parca-agent
+
+          # Set some options to allow retries while Parca comes back up
+          CURL_OPTS=(--max-time 20 --retry 5 --retry-delay 3 --retry-connrefused)
+
+          curl ${CURL_OPTS[@]} http://localhost:8081/
+          curl ${CURL_OPTS[@]} http://localhost:8081/metrics
+
+      # In case the above tests fail, dump the logs for inspection
+      - name: Dump snap service logs
+        if: failure()
+        run: |
+          sudo snap logs parca-agent -n=all
+
+  release-snap-edge:
+    name: Release Snap (latest/edge)
+    needs: test-snap
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: built-snaps
+
+      - name: Install snapcraft
+        run: |
+          sudo snap install snapcraft --classic --channel=8.x/stable
+
+      - name: Release to latest/edge
+        env:
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_STORE_CREDENTIALS }}
```

Comment:
We need to make sure this is still set - though I'm guessing it's probably still in the secrets config for the repo

Reply:
yep, it is!